### DISCUSSION TITLE
Webhook Notifications

### DISCUSSION THREAD

**Discussion by silasabbott** [coollabsio/coolify#1602] [Feature Requests]
We got Email, Telegram, and Discord notifications for service status changes (under team settings). Adding in webhooks there as well would be amazing!

**Comment by sivadass** [comment]
It would be really great for the people who don't use Telegram or Discord.

**Comment by ohld** [comment]
UP!

There is also a need to get the deployment logs without having to login to Coolify instance.

Our project setup is follows: only a devop have the access to Coolify instance. The dev team simply push commits to github. I'd like the dev team to be able to see the deployment logs like they used to when we used Github Actions.

**Reply by HammyHavoc** [reply]
Add a feature request for this if you haven't already, or it'll get lost in the comments on this irrelevant issue.

**Comment by mredodos** [comment]
+1 With webhook (similar to discord) nitification you can litarally connect what you want

**Comment by gKits** [comment]
+1 Beeing able to use a custom notification service like gotify or ntfy would be a game changer

**Comment by Tam2** [comment]
+1 we need this to run various automations post deployment (update DNS automatically, sync with external reverse proxy etc)

Due to multi server setup we need this to keep on top of keeping config in sync

Edit: we're using the discord ones for this, as discord notification setup accepts a webhook url

**Comment by sofianegargouri** [comment]
Webhooks is now implemented, but it would be nice to be able to add multiple configurations.
